components/expand-collapse/README.md

```diff
@@ -36,7 +36,12 @@ The `d2l-expand-collapse-content` element can be used to create expandable and c
 - `d2l-expand-collapse-content-collapse`: dispatched when the content starts to collapse. The `detail` contains a `collapseComplete` promise that can be waited on to determine when the content has finished collapsing.
 <!-- docs: end hidden content -->
 
-### Accessibility Properties
+## Accessibility
 
-To make your usage of `d2l-expand-collapse-content` accessible, the [`aria-expanded` attribute](https://www.w3.org/TR/wai-aria/#aria-expanded) should be added to the element that controls expanding and collapsing the content with `"true"` or `"false"` to indicate that the content is expanded or collapsed.
+To make your usage of `d2l-expand-collapse-content` accessible, it should follow the [W3C Disclosure (Show/Hide) Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/).
+
+To achieve this, the control that toggles the expanded state should:
+ - Have the [`button`](https://w3c.github.io/aria/#button) role
```

Comment:
Should we somehow indicate that the `button` within `d2l-button` would already have the `button` role just so it doesn't end up getting added to the `d2l-button`?

Reply:
I could switch it to be:

> Use the [`d2l-button`](../buttons) component or have the [`button`](https://w3c.github.io/aria/#button) role

Thoughts?

Reply:
I think that's a good improvement, yep!